packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -140,13 +140,13 @@ const useTrackDesignView = ( {
 
 				recordTracksEvent( 'calypso_design_picker_design_display', {
 					category: trackingCategory,
-					design_type: design.design_type,
 					...( design?.design_tier && { design_tier: design.design_tier } ),
-					is_premium: design.is_premium,
+					is_premium: design?.design_tier === 'premium',
+					// TODO: Better to track whether already available on this sites plan.
 					is_premium_available: isPremiumThemeAvailable,
 					slug: design.slug,
 					is_virtual: design.is_virtual,
-					is_externally_managed: design.is_externally_managed,
+					is_externally_managed: design?.design_tier === 'partner',
```

Comment:
Do we need this prop? I guess that we can infer this from the actual design tier. Nevermind, we still have some themes that don't have a tier assigned.

Reply:
The goal is to remove `is_externally_managed` from the design object. I've left the `is_externally_managed` track prop in so that we can still compare data from before theme tiers was introduced.